carta/image.py

```diff
@@ -593,6 +593,87 @@ def hide_contours(self):
         """Hide the contours."""
         self.set_contours_visible(False)
 
+    # VECTOR OVERLAY
+
+    @validate(Constant(VectorOverlaySource), Constant(VectorOverlaySource), NoneOr(Number()), Boolean(), NoneOr(Number()), Boolean(), NoneOr(Number()), NoneOr(Number()))
+    def configure_vector_overlay(self, angular_source=VectorOverlaySource.CURRENT, intensity_source=VectorOverlaySource.CURRENT, pixel_averaging=4, fractional_intensity=False, threshold=None, debiasing=False, qError=None, uError=None):
+        """Configure vector overlay.
+
+        Parameters
+        ----------
+        angular_source : {0}
+            The angular source.
+        intensity_source : {1}
+            The intensity source.
+        pixel_averaging : {2}
+            The pixel averaging width. Set it to None to disable the usage pixel averaging width.
+        fractional_intensity : {3}
+            Set polarization intensity to absolute or fractional.
+        threshold : {4}
+            The threshold. Set it to None to disable the usage of threshold.
```

Comment:
I would say: The threshold. By default the threshold is disabled.

(We should probably also mention the units here.)

(If the user doesn't want to enable the threshold, they are more likely to omit the parameter and fall back to the default than to pass in an explicit `None`.)